### PR TITLE
CSS hyphens example

### PR DESCRIPTION
**Pull request by jpmedley**

### CODE REVIEW

File: css-hyphens/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<!-- TODO: Replace PLACEHOLDER with feature name. -->
```
Comment (jeffposnick) [inline css-hyphens/README.md]: Can you remove this and the other comments from the file?

File: css-hyphens/index.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+<p>In the second example, the <code>hyphens</code> property is set to <code>manual</code> meaning the soft hyphen will only appear when the margin breaks the word 'elit'. Again, you can confirm this by adjusting the window size.</p>
+
+<p>In the third example, the <code>hyphens</code> property is set to <code>auto</code>. In this case, no soft hyphen is needed since the user agent will determine hyphen locations automatically. If you resize the window, you'll see that the browser hyphenates the third example in the same place as in the second example, though no soft hyphen is present. If you continue to shrink the window, you'll see that your browser is able to break lines between any two syllables in the text.<p>
+
+{% capture css %}
+div {
+  font: 18px Times;
```
Comment (jeffposnick) [inline css-hyphens/index.html]: I don't know how widely available `Times` is on different platforms. Maybe just go with `serif` here?

**Review by jeffposnick — CHANGES_REQUESTED** [review]

**Comment by jpmedley** [comment]
@jeffposnick PTAL

**Comment by jeffposnick** [comment]
👍

**Pull request merged**